pkg/agent/cniserver/ipam/annotations.go

```diff
@@ -15,6 +15,9 @@
 package ipam
 
 const (
-	AntreaIPAMAnnotationKey       = "ipam.antrea.io/ippools"
-	AntreaIPAMAnnotationDelimiter = ","
+	AntreaIPAMAnnotationKey            = "ipam.antrea.io/ippools"
```

Comment:
Add some comments to describe where the annotations should be added, inc. the considerations for StatefulSet and Deployment.

Reply:
Sure.

Reply:
Added comments here, and will describe this in doc.